packages/centreon-ui/src/InputField/Select/Autocomplete/index.tsx

```diff
@@ -209,7 +209,9 @@ const AutocompleteField = ({
         root: classes.textfield,
       }}
       forcePopupIcon={displayPopupIcon}
-      getOptionLabel={(option: SelectEntry): string => option.name}
+      getOptionLabel={(option: string | SelectEntry): string =>
+        typeof option === 'string' ? option : option.name
```

Comment:
```suggestion
        is(String, option) ? option : option.name
```
`is` comes from Ramda https://ramdajs.com/docs/#is

Reply:
In what  is the getOptionLabel a simple string instead of an object ?